plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/command/RemoveConnectionHandler.java

```diff
@@ -31,6 +31,9 @@
 	@Override
 	public Object execute(ExecutionEvent event) throws ExecutionException {
 		IConnection[] connections = UIUtils.getElements(HandlerUtil.getCurrentSelection(event), IConnection.class);
+		if( connections.length == 0 )
+			return null;
```

Comment:
null connections should be filtered out.

If resulting list is empty, then return